clang/lib/CodeGen/CodeGenFunction.cpp

```diff
@@ -3041,3 +3046,20 @@ llvm::Value *CodeGenFunction::emitBoolVecConversion(llvm::Value *SrcVec,
 
   return Builder.CreateShuffleVector(SrcVec, ShuffleMask, Name);
 }
+
+void CodeGenFunction::EmitPointerAuthOperandBundle(
```

Comment:
`PascalCase` for variables in this function - surrounding code already uses this naming convention